pkg/proc/breakpoints.go

```diff
@@ -79,6 +79,9 @@ const (
 	// Continue will set a new breakpoint (of NextBreakpoint kind) on the
 	// destination of CALL, delete this breakpoint and then continue again
 	StepBreakpoint
+	// TraceReturnBreakpoint is an internal breakpoint set to get the return
+	// values of a traced function.
+	TraceReturnBreakpoint
```

Comment:
I don't think you are using this anywhere.

Reply:
Meant to remove this before pushing.